### ISSUE TITLE
link for icon table component

### ISSUE THREAD

**Issue by f8dca** [lovasoa/SQLpage#56]
I would like to have the capability of an icon with a clickable link in table component line.

**Comment by lovasoa** [comment]
Hi @f8dca !

It's currently not supported directly by SQLPage's icon system,
but you can get the result you want using a markdown column for the icon and the link : 

```sql
select 'table' as component,
    'My icon' AS markdown;

select '[
    ![](https://tabler-icons.io/static/tabler-icons/icons-png/accessible-off-filled.png)
](https://google.com)' as "My icon",
    'John' as "First Name",
    'Doe' as "Last Name",
    1994 as "Birth Date";
```

![screenshot sqlpage icon link](https://github.com/lovasoa/SQLpage/assets/552629/66d05728-cf1a-4c5f-b593-811cd6982b59)

Does that do what you need ?

**Comment by f8dca** [comment]
Yes, i can do this but you should aim for consistency. Other components already have the link attribute so it would make sense to add it to the table too.
